casepro/pods/urls.py

```diff
@@ -0,0 +1,8 @@
+from django.conf.urls import url
+
+from casepro.pods.views import perform_pod_action, read_pod_data
+
+urlpatterns = (
+    url(r'read/(?P<pk>\d+)/$', read_pod_data, name='read_pod_data'),
```

Comment:
OK I was thinking that pod templates might need to manually reference these URLs but clearly that's not where you're going with these.